nvme-print-stdout.c

```diff
@@ -246,15 +246,14 @@ static void stdout_add_bitmap(int i, __u8 seb)
 	}
 }
 
-static void stdout_persistent_event_log_fdp_events(unsigned int cdw11,
-						   unsigned int cdw12,
-						   unsigned char *buf)
+static void stdout_persistent_event_log_fdp_events(struct nvme_feat_fdp_events_cdw11 *cdw11,
+						   unsigned int cdw12, unsigned char *buf)
```

Comment:
Uhh, didn't know there was `struct nvme_feat_fdp_events_cwd11` although I merged it...

I don't think we should have this struct around in the first place. This is a CWD thus we should have just SHIFTS and MASKS definition as we have for all other commands. And the you can use NVME_GET(cdw11, ...) as you do for the cwd12.

And please mark `struct nvme_feat_fdp_events_cwd11` as deprecated. 

@birkelund

Reply:
For the review comment just created the libnvme PR https://github.com/linux-nvme/libnvme/pull/940.

Reply:
Just fixed the patch to use NVME_GET instead. Also rebased with the PR #2650 changes since the libnvme bump required and the build warings errors caused without the PR changes. Thank you.